### DISCUSSION TITLE
Blog post are not ordered by date

### DISCUSSION THREAD

**Discussion by papanito** [thuliteio/doks#1296] [Q&A] [answered]
When I navigate to ` /blogs`  the lust of blogs is not in chronological order. I have the same already on my current blog, which uses an old version of your theme (plus modifications)

https://wyssmann.com/blog/

**Comment by h-enk** [comment, marked as answer]
https://wyssmann.com/blog/ looks ordered date descending to me?


What if you update line 10 of [`layouts/_default/list.html`](https://github.com/thuliteio/doks/blob/v0.5.0/layouts/_default/list.html#L10)?

From:

```html
{{ range $paginator.Pages -}}
```

To:

```html
{{ range $paginator.Pages.ByDate -}}
```

**Reply by papanito** [reply]
Adding `.ByDate`  order them but first one first. How can I reverse the order?

**Reply by papanito** [reply]
Ahh `.ByDate.Reverse` does the trick. Many thanks
